packages/progress-indicator/src/LionProgressIndicator.js

```diff
@@ -0,0 +1,110 @@
+import { svg, css, LitElement } from '@lion/core';
+import { localize, LocalizeMixin } from '@lion/localize';
+
+export class LionProgressIndicator extends LocalizeMixin(LitElement) {
+  static get styles() {
+    return [
+      css`
+        :host {
+          display: inline-block;
+        }
+
+        svg.lion-progress {
+          animation: progress-indicator-rotate 2s linear infinite;
+          height: 40px;
+          width: 40px;
+        }
+
+        @keyframes progress-indicator-rotate {
+          to {
+            transform: rotate(360deg);
+          }
+        }
+      `,
+    ];
+  }
+
+  static get localizeNamespaces() {
+    return [
+      {
+        'lion-loading-indicator': locale => {
+          switch (locale) {
+            case 'bg-BG':
+            case 'bg':
+              return import('../translations/bg.js');
+            case 'cs-CZ':
+            case 'cs':
+              return import('../translations/cs.js');
+            case 'de-DE':
+            case 'de':
+              return import('../translations/de.js');
+            case 'en-AU':
+            case 'en-GB':
+            case 'en-US':
+            case 'en-PH':
+            case 'en':
+              return import('../translations/en.js');
+            case 'es-ES':
+            case 'es':
+              return import('../translations/es.js');
+            case 'fr-BE':
+            case 'fr-FR':
+            case 'fr':
+              return import('../translations/fr.js');
+            case 'hu-HU':
+            case 'hu':
+              return import('../translations/hu.js');
+            case 'it-IT':
+            case 'it':
+              return import('../translations/it.js');
+            case 'nl-BE':
+            case 'nl-NL':
+            case 'nl':
+              return import('../translations/nl.js');
+            case 'pl-PL':
+            case 'pl':
+              return import('../translations/pl.js');
+            case 'ro-RO':
+            case 'ro':
+              return import('../translations/ro.js');
+            case 'ru-RU':
+            case 'ru':
+              return import('../translations/ru.js');
+            case 'sk-SK':
+            case 'sk':
+              return import('./translations/sk.js');
+            case 'uk-UA':
+            case 'uk':
+              return import('../translations/uk.js');
+            default:
+              return import('../translations/en.js');
+          }
+        },
+      },
+    ];
+  }
+
+  // eslint-disable-next-line class-methods-use-this
+  template() {
+    return svg`
+      <svg class="lion-progress" viewBox="22 22 44 44">
+        <circle cx="44" cy="44" r="20.2" fill="none" stroke-width="3.6" stroke="black" stroke-dasharray="88,25" />
+      </svg>
+    `;
+  }
+
+  render() {
+    return this.template();
+  }
+
+  connectedCallback() {
+    super.connectedCallback();
+    this.setAttribute('role', 'status');
```

Comment:
so we decided not to go for https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_progressbar_role?

Reply:
It probably will be needed if we want to make this the basis for a linear progress indicator as well and if the progress needs to have a concrete value

Reply:
I think the progressbar role only applies to determinate progress indicators. This implementation is the basis for indeterminate progress indicators, and we could provide a basis for a determinate indicator as an extension of this. We could also decide later to extend this and provide explicit support for determinate indicators.

Do you think this is an ok way forward?

Reply:
After our discussion last week, I was under the impression that we would build a component that lays the foundation for all appearances as listed here: https://material-ui.com/components/progress/
In that case, it should probably be a role=progressbar.

In case of only supporting indetermate indicators, this might be sufficient.
Maybe we should do a follow up story then with a good a11y assessment and cross browser tests....

Reply:
We could add a `determinate` attribute or something similar, and have the internal behaviour set the correct aria attributes based on the chosen mode. Always setting role=progressbar will cause problems for non-determinate indicators, so I think having the current behaviour as a default will be fine. We can add additional support for determinate indicators later in a non-breaking change.

Reply:
I think it's possible to have indeterminate progressbars as well: 
- https://material-ui.com/components/progress/#circular

- https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_progressbar_role ->
> If the progress value  is indeterminate, the developer should omit the aria-valuenow attribute.

Reply:
For now we can probably leave at is and later extend a11y, without breaking the public api.